static/scss/email-composition-dialog.scss

```diff
@@ -131,6 +131,10 @@
     }
   }
 
+  .send-one-time-email {
```

Comment:
Can you add this as a rule under `.type-radio-group` above, and also apply it to all child divs instead of just `send-one-time-email`?